cmd/torusd/main.go

```diff
@@ -262,3 +255,11 @@ func doAutojoin(s *torus.Server) error {
 		return err
 	}
 }
+
+func addrToUri(addr string) (*url.URL, error) {
```

Comment:
not a fan of moving util code to cmd main.go.  but we do not have any internal util pkg now.

opinions? @barakmich @lpabon